src/index.js

```diff
@@ -1,9 +1,22 @@
 
 var fs = require('fs');
 var AWS = require('aws-sdk');
+//var mysql = require('mysql');
+var sleep = require('sleep');
+const McawsModels = require('./models/mcawsModels.js');
 
 var baseHandler = require('aws-services-lib/lambda/base_handler.js')
 
+function getRulesPayload(rule){
+  return new Promise(function(resolve,reject) {
+    if(rule.owner == "CUSTOM_LAMBDA"){
```

Comment:
What if the owner attribute is not present?

Reply:
as of now its not using anywhere

Reply:
do you mean getRulesPayload is not used anywhere?

Reply:
ignore previous comment 
master account id require to set only with custom config rule

Reply:
For case of custom config rule, we need to handle it

Reply:
If role owner is not "CUSTOM_LAMBDA" then no need set any thing that's why i did not put any else part